frontend/storybook/src/components/entry/imagelabel/one.ts

```diff
@@ -0,0 +1,38 @@
+import "@elements/entry/admin/images/meta/page";
+import { LeftLabel } from "~/components/entry/imagelabel/admin/images/image-label-left";
+import {LabelRight} from "~/components/entry/imagelabel/admin/images/image-label-right";
+import "@elements/core/titles/variants/underlined-title";
+import { Rectangle } from "~/components/core/buttons/rectangle";
+
+export default {
+  title: 'Full Pages/Image Label',
+}
+
+interface Args {
+}
+
+const DEFAULT_ARGS:Args = {
+}
+
+
+const STR_NEXT = "Next";
+const STR_LABEL ="Label Images"
+
+export const ImageLabelFullOne = (props?:Args) => {
```

Comment:
Args should be `Partial<Args>` see:
https://docs.sandbox.jicloud.org/book/frontend/elements-and-components/guideline-components.html#provide-arguments